webapp/src/components/Modals/ConfirmRentModal/ConfirmRentModal.module.css

```
@@ -56,3 +56,21 @@
 .signMessage {
   margin-left: 12px;
 }
+
+@media (max-width: 700px) {
+  .modal:global(.ui.modal > .actions) {
+    align-items: center;
+    bottom: 28px;
+    margin: 0px !important;
```

I've removed the important and it seems to work fine:
```suggestion
    margin: 0px;
```